uat-web-app/static/js/sorting/main.js

```diff
@@ -202,6 +227,7 @@ class NewBranchEntry {
         $("#add_node").click(function () {
             AddNode();
             $("#newnode").val("");
+            $('#add_node').prop('disabled', true);
```

Comment:
For the `#` calls I would standardize around either `'` or `"` and make sure it is consistent across the file.